Code-the-Dream-School/ee-prac-team3-front
# Feature/quiz question UI

## Purpose: 

Integrate functional quiz component into current working UI

## Summary:

- Created routes for quizes based on quizID 
- Made some suggested changes from previous PR and additionally added functionality changes to provide user feedback after answering questions. 
- linked quiz UI to quiz cards (clicking any quiz card leads to quiz UI with dummy data)
- added conditional rendering to next question and previous question buttons
- added "Submit Quiz" button that is visible after completing a quiz
- added dialogue boxes to give user feedback when skipping a question or submitting a question or quiz 

## Front End Change:

App.jsx
QuestionContent.jsx
QuizContent.jsx
package.json{}

## Test Steps:

From homepage, sign in and click the "Quizzes" button.  Click any quiz card to take you to dummy "addition" quiz.  Feel free to play with answering the questions correctly or incorrectly.  UI should give feedback.  User should be able to submit quiz if user has navigated to the last question or answered all questions.

## Note on styling

Component is not properly styled.  Kat has offered to help with styling as she understands the styling of the app better than I do. Thanks Kat!
